driver/API_VERSION

```diff
@@ -1 +1 @@
-7.0.0
+8.0.0
```

Comment:
This has been bumped recently. Will let reviewers decide if the extra bump is appropriate.

Reply:
https://github.com/falcosecurity/libs/blob/master/driver/README.VERSION.md#driver-internal-versioning
> The version numbers described below must be incremented every time and only when a single change or an atomic group of changes - which meet the criteria described in the relative When to Increment section below - is included in the master branch. Thus, a version bump can occur multiple times during the development and testing phases of a given release cycle. A given version bump must not group multiple changes that occurred sporadically during the release cycle.

If a change is introduced by this PR, it must be bumped.

Reply:
Done then!